assets/src/edit-story/components/panels/textStyle/textStyle.js

```diff
@@ -128,10 +128,12 @@ function StylePanel({ selectedElements, pushUpdate }) {
         />
         <ToggleButton
           icon={<BoldIcon />}
-          value={bold === true}
+          value={fontWeight === 700}
```

Comment:
I think this has to be 700+. If the user has selected 800 or 900. bold should still appear as toggled.